pandas/core/arrays/string_arrow.py

```diff
@@ -178,6 +184,10 @@ def insert(self, loc: int, item) -> ArrowStringArray:
             raise TypeError("Scalar must be NA or str")
         return super().insert(loc, item)
 
+    @staticmethod
+    def _result_converter(values, **kwargs):
```

Comment:
in the subclass kwargs is just `na`.  can that be done here too?

Reply:
Yep, done